flare/qe_util.py

```diff
@@ -24,6 +24,8 @@ def run_espresso_par(qe_input, structure, pw_loc, no_cpus):
         'mpirun -np {0} {1} < {2} > {3}'.format(no_cpus, pw_loc, run_qe_path,
                                                 'pwscf.out')
 
+    with open("alog", "a+") as fout:
```

Comment:
What does alog mean / stand for?

Reply:
ah... It's a line I left for debug. I forgot to delete it...